### PR TITLE
Update GaTech LIGO cluster info

### PR DESCRIPTION
**Pull request by sthapa**

### CODE REVIEW

File: topology/Georgia Institute of Technology/Georgia Tech/Georgia Tech PACE.yaml [hunk 1]
```diff
@@ -132,7 +132,7 @@ Resources:
           Name: Peter Couvares
     Description: Squid server for PACE.
     Disable: false
-    FQDN: ligo-prx.pace.gatech.edu
+    FQDN: osg-sched.pace.gatech.edu
```
[the remark this answers is not in the document]
Comment (sthapa) [reply topology/Georgia Institute of Technology/Georgia Tech/Georgia Tech PACE.yaml]: No, the gatech pace cluster only has 4 server nodes so we're running multiple services on the servers.  The CE/condor collector/negotiator/squid services are all on that node.

[the remark this answers is not in the document]
Comment (brianhlin) [reply topology/Georgia Institute of Technology/Georgia Tech/Georgia Tech PACE.yaml]: In that case, it sounds like we have a single resource `osg-sched.pace.gatech.edu` with multiple services.

(CC @edquist @efajardo @timtheisen)

[the remark this answers is not in the document]
Comment (matyasselmeci) [reply topology/Georgia Institute of Technology/Georgia Tech/Georgia Tech PACE.yaml]: In this case it looks like two resources (two machines) are being combined into one now. What should we do with `Georgia_Tech_PACE_Squid_1`? Move the Squid service, delete the resource, and all the downtime registered for it?

[the remark this answers is not in the document]
Comment (brianhlin) [reply topology/Georgia Institute of Technology/Georgia Tech/Georgia Tech PACE.yaml]: Yeah, that sounds like a good solution to me!

[the remark this answers is not in the document]
Comment (sthapa) [reply topology/Georgia Institute of Technology/Georgia Tech/Georgia Tech PACE.yaml]: Is there anything I need to do in order to get this merged?

**Pull request merged**